avm/src/daemon/ctrl.rs

```diff
@@ -150,6 +157,12 @@ async fn control_port(req: Request<Body>) -> Result<Response<Body>, Infallible>
     "/health" => Ok(Response::builder().status(200).body("ok".into()).unwrap()),
     "/clusterHealth" => handle_cluster_health(),
     "/start" => handle_start(req).await,
+    "/datastore/getf" => handle_dsgetf(req).await, // TODO: How to better organize the datastore stuff?
+    "/datastore/getv" => handle_dsgetv(req).await,
```

Comment:
What does the `f` and the `v` stand for?

Reply:
`f`ixed and `v`ariable. Basically is the value (stored as) a single 64-bit integer, or is it an array of some arbitrary size and shape?